src/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixer.php

```diff
@@ -76,14 +73,50 @@ protected function createConfigurationDefinition()
                     ->setAllowedTypes(['bool'])
                     ->setDefault(true)
                     ->getOption(),
-                (new FixerOptionBuilder('around_argument_assignments', 'Whether to fix spaces around argument assignment operator.'))
+                (new FixerOptionBuilder('around_argument_assignments', 'Whether to fix spaces around argument assignment operator (deprecated, use `before_argument_assignments` and `after_argument_assignments` options instead).'))
                     ->setAllowedTypes(['bool'])
                     ->setDefault(true)
                     ->getOption(),
-                (new FixerOptionBuilder('around_array_assignments', 'Whether to fix spaces around array assignment operators.'))
+                (new FixerOptionBuilder('before_argument_assignments', 'Whether to add, remove or ignore spaces before argument assignment operator.'))
+                    ->setAllowedTypes(['null', 'bool'])
+                    ->setDefault(function (Options $options) {
+                        return $options['around_argument_assignments'] ? false : null;
```

Comment:
This is the BC part: using the deprecated options will not directly tweak fixer's behavior anymore but will set the default value of the new options, which then tweak the fixer's behavior.